rskj-core/src/test/java/co/rsk/peg/BridgeStorageProviderTest.java

```diff
@@ -97,7 +97,7 @@ void createInstance() throws IOException {
         BridgeStorageProvider provider = new BridgeStorageProvider(
             repository,
             PrecompiledContracts.BRIDGE_ADDR,
-            config.getNetworkConstants().getBridgeConstants(),
+            BridgeTestNetConstants.getInstance(),
```

Comment:
Why not simple put this in a variable at class level and use it everywhere? Like networkParameters